erpnext/stock/doctype/stock_settings/stock_settings.json

```diff
@@ -420,6 +421,13 @@
    "fieldname": "auto_reserve_stock_for_sales_order_on_purchase",
    "fieldtype": "Check",
    "label": "Auto Reserve Stock for Sales Order on Purchase"
+  },
+  {
+   "default": "0",
+   "description": "Allows duplicacy across separate items. Maintains uniqeness within an item.",
```

Comment:
- In standard English, unfortunately there only seems to be the word “duplication,” not “[duplicacy](https://en.wiktionary.org/wiki/duplicacy)”, though I personally like it.
- “uniq**u**eness”
- Also, we need to explain what may be duplicated.

Propose: “Allow duplicate serial numbers across different Items. Maintain uniqueness per Item.”

Reply:
That does sound better. Leave a direct code review next time, quicker to commit and also co-author you

Reply:
Sry, I would if I could. But the Code review feature isn't available to me as a normal contributor... 💁🏻‍♂️